### DISCUSSION TITLE
Why would one record the masterkey?

### DISCUSSION THREAD

**Discussion by ghost** [rfjakob/gocryptfs#652] [Q&A] [answered]
gocryptfs recommends writing down the masterkey. Wouldn't it be simpler to just write down the actual password? Does this have any drawbacks?

**Comment by rfjakob** [comment, marked as answer]
The advantages of storing the masterkey are:
1) It never changes, even if the password is changed
2) It works even if you lose the file `gocryptfs.conf`
